### PR TITLE
Blog: ZAP SSRF setup

### PR DESCRIPTION
**Pull request by DiogoMRSilva**
Planned release date: March 9th

Signed-off-by: Diogo Silva <diogo.m.r.silva@tecnico.ulisboa.pt>

### CODE REVIEW

File: site/content/blog/2020-03-09-zap-ssrf-setup/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    ```sh
+    ssh -N -R 38193:localhost:38193 myUser@something.mydomain.org
+    ```
+
+If you want to have a valid HTTPS certificate you can run some reverse proxy on your server (as Nginx) with a certificate, proxy the requests to other port, for instance 9999, and change the port forwarding to that address:
+```sh
+ssh -N -R 9999:localhost:38193 myUser@something.mydomain.org
+```
+
+__Troubleshot__:
```
Comment (kingthorin) [inline site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: Make it a heading instead if bold?
Two Os in shoot 👍

**Comment by psiinon** [comment]
Ooops - missed the Troubleshot comment :) @DiogoMRSilva can you submit a PR with that tweak? Then I think we'll be ready to publish?

File: site/content/blog/2020-03-09-zap-ssrf-setup/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- make shure you allow port forwarding on the server ssh config `/etc/ssh/sshd_config`:
+    ```
+    ...
+    AllowTcpForwarding yes
+    GatewayPorts yes
+    ...
+    ```
+
+## Conclusion
+While other vulnerability scanners like Burp Suite have an external server ([Collaborator](https://portswigger.net/burp/documentation/collaborator)) which facilitate reception of requests from the “victim” server, ZAP still lacks that functionality. Luckily it can be obtained with this easy trick. This solution still lacks the ability to log DNS requests that can be used for out of band (OOB) tests, which may be necessary when the “victim” app/system blocks outbound HTTP/HTTPS connections.
```
Comment (thc202) [inline site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: `"victim"` → `target`?

File: site/content/blog/2020-03-09-zap-ssrf-setup/index.md (hunk 1 above)
Comment (DiogoMRSilva) [reply site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: like ###Troubleshot ?

Comment (kingthorin) [reply site/content/blog/2020-03-09-zap-ssrf-setup/index.md]: `### Troubleshoot` I think

**Pull request merged**